### PR TITLE
Update deps

### LINKED ISSUE #68: Update dependencies
See https://david-dm.org/ckeditor/ckeditor5-angular?type=dev

### PR DESCRIPTION
**Pull request by ma2ciek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Updated dependencies. Closes #68.

---

### Additional information

*For example – encountered issues, assumptions you had to make, other affected tickets, etc.*

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,33 +30,33 @@
...
-    "@types/jasmine": "^2.8.8",
+    "@types/jasmine": "^3.3.8",
     "@types/jasminewd2": "^2.0.3",
     "codelyzer": "^4.4.2",
     "coveralls": "^3.0.2",
     "fs-extra": "^7.0.0",
     "jasmine-core": "^3.2.1",
     "jasmine-spec-reporter": "^4.2.1",
-    "karma": "^3.0.0",
+    "karma": "^4.0.0",
```
Comment (pomek) [inline package.json]: If you want to bump the Karma version, you need to bump supported Node version (`engines.node`). See https://github.com/karma-runner/karma/issues/3151.

cc: @Reinmar

Comment (ma2ciek) [reply package.json]: On the one hand, it's mainly for internal usage. On the other hand, I've just installed the latest angular sample app and it's using `karma@3.1.1`. Do we have somewhere define when we drop node@6 support?

Comment (pomek) [reply package.json]: No, I don't. I've just created a ticket about bumping the Node version (https://github.com/ckeditor/ckeditor5/issues/1507).

Comment (ma2ciek) [reply package.json]: I see from the official nodejs page (https://github.com/nodejs/Release), that they will drop support for node@6 in April 2019.

**Comment by pomek** [comment]
R- until resolving the node version issue (https://github.com/ckeditor/ckeditor5-angular/pull/69#discussion_r253790649).

**Comment by pomek** [comment]
I'll bump versions of Node and NPM.

**Pull request merged**